data_steward/retraction/retract_deactivated_pids.py

```diff
@@ -43,15 +43,10 @@
 {% if has_mapping or has_ext %}
 LEFT JOIN `{{mapping_ext_ref.project}}.{{mapping_ext_ref.dataset_id}}.{{mapping_ext_ref.table_id}}` m
 USING ({{table_id}})
-{% if has_mapping %}
-WHERE src_hpo_id != 'PPI/PM'
-{% elif has_ext %}
-WHERE src_id != 'PPI/PM'
-{% endif %}
 {% endif %}
```

Comment:
Even though we don't need this, there is a runtime error being raised if an ext/mapping table is not found right now, saying 'EHR data cannot be identified'. The whole section relating to ext/mapping can be removed.